service/matching/matching_engine.go

```diff
@@ -1958,7 +1990,7 @@ func (e *matchingEngineImpl) pollTask(
 }
 
 // Unloads the given task queue partition. If it has already been unloaded (i.e. it's not present in the loaded
-// partitions map), then does nothing.
+// partitions map), unloadPM.Stop(...) is still called.
```

Comment:
This is being removed in another PR because it's actually a bug, so please don't change this comment.
Actually we should have done just that in a tiny pr :(

Reply:
Please change this back

Reply:
Sorry, snuck in from a stash/old commit